cmssw-tools.spec

```diff
@@ -146,6 +146,7 @@ Requires: dasgoclient
 Requires: dablooms
 Requires: zlib
 Requires: rivet
+Requires: clue
```

Comment:
I don't remember why `alpaka` is in the Linux-only section, but could you move `clue` to the same spot, right after `alpaka` itself ?

Reply:
@fwyzard you are right, since clue depends on alpaka.
Did it, thanks.